invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposits/RDMDepositForm/RDMDepositForm.js

```diff
@@ -101,20 +113,79 @@ export class RDMDepositForm extends Component {
+        },
+        related_identifiers: {
+          resource_type: this.config.vocabularies.resource_type,
+          scheme: [
+            { text: "ARK", value: "ark" },
+            { text: "ARXIV", value: "arxiv" },
+            { text: "BIBCODE", value: "bibcode" },
+            { text: "DOI", value: "doi" },
+            { text: "EAN13", value: "ean13" },
+            { text: "EISSN", value: "eissn" },
+            { text: "HANDLE", value: "handle" },
+            { text: "IGSN", value: "igsn" },
+            { text: "ISBN", value: "isbn" },
+            { text: "ISSN", value: "issn" },
+            { text: "ISTC", value: "istc" },
+            { text: "LISSN", value: "lissn" },
+            { text: "LSID", value: "lsid" },
+            { text: "PMID", value: "pmid" },
+            { text: "PURL", value: "purl" },
+            { text: "UPC", value: "upc" },
+            { text: "URL", value: "url" },
+            { text: "URN", value: "urn" },
+            { text: "W3ID", value: "w3id" },
+          ],
+          relations: [
+            { text: "ISCITEDBY", value: "iscitedby" },
+            { text: "CITES", value: "cites" },
+            { text: "ISSUPPLEMENTTO", value: "issupplementto" },
+            { text: "ISSUPPLEMENTEDBY", value: "issupplementedby" },
+            { text: "ISCONTINUEDBY", value: "iscontinuedby" },
+            { text: "CONTINUES", value: "continues" },
+            { text: "ISDESCRIBEDBY", value: "isdescribedby" },
+            { text: "DESCRIBES", value: "describes" },
+            { text: "HASMETADATA", value: "hasmetadata" },
+            { text: "ISMETADATAFOR", value: "ismetadatafor" },
+            { text: "HASVERSION", value: "hasversion" },
+            { text: "ISVERSIONOF", value: "isversionof" },
+            { text: "ISNEWVERSIONOF", value: "isnewversionof" },
+            { text: "ISPREVIOUSVERSIONOF", value: "ispreviousversionof" },
+            { text: "ISPARTOF", value: "ispartof" },
+            { text: "HASPART", value: "haspart" },
+            { text: "ISREFERENCEDBY", value: "isreferencedby" },
+            { text: "REFERENCES", value: "references" },
+            { text: "ISDOCUMENTEDBY", value: "isdocumentedby" },
+            { text: "DOCUMENTS", value: "documents" },
+            { text: "ISCOMPILEDBY", value: "iscompiledby" },
+            { text: "COMPILES", value: "compiles" },
+            { text: "ISVARIANTFORMOF", value: "isvariantformof" },
+            { text: "ISORIGINALFORMOF", value: "isoriginalformof" },
+            { text: "ISIDENTICALTO", value: "isidenticalto" },
+            { text: "ISREVIEWEDBY", value: "isreviewedby" },
+            { text: "REVIEWS", value: "reviews" },
+            { text: "ISDERIVEDFROM", value: "isderivedfrom" },
+            { text: "ISSOURCEOF", value: "issourceof" },
+            { text: "ISREQUIREDBY", value: "isrequiredby" },
+            { text: "REQUIRES", value: "requires" },
+            { text: "ISOBSOLETEDBY", value: "isobsoletedby" },
```

Comment:
Do we want spaces between the `text` values e.g. `ISOBSOLETEDBY` -> `Is obsoleted by` or are curators more familiar with the all caps together?

Reply:
Question: ``text`` is what is displayed to the user? if so, then yes it should be changed.

Reply:
Changed to human-readable format 😇